guide/additional-info/async-await.md

```diff
@@ -35,11 +35,28 @@ deleteMessages(5).then(value => {
 });
 ```
 
-In this scenario the `deleteMessages` function returns a Promise. The `.then()` function will trigger if the Promise was resolved and the `.catch()` function if the Promise was rejected. But with our function we resolve the Promise after 2 seconds with the String "Everything is fine", so the `.catch()` function will never be executed.
+In this scenario the `deleteMessages` function returns a Promise. The `.then()` method will trigger if the Promise was resolved and the `.catch()` method if the Promise was rejected. But with our function we resolve the Promise after 2 seconds with the String "Everything is fine", so the `.catch()` method will never be executed.
 
 ## How to implement async/await
 
-After knowing how Promises works and what they are for, let's look at an example in which we handle multiple Promises. Let's say you wanted to react to a message in a certain order with letters (regional indicators). For this example, I will take the basic template for a discord.js bot with some ES6 adjustments.
+### Theory
+The following Information are important to know before starting working with async/await. You can only use the await keyword inside a function that is declared as async (you put the async keyword before the  function keyword or the parameter when you use an arrow function) a simple example would be:
+```js
+async function markedAsAsync() {
+	// code
+}
+``` 
+or 
+```js 
+async () => {
+	// code
+}
+```
+Also an important thing to know, is that a function declared as async will always return a Promise. In addition to this, if you return something inside a async function the returned promise will resolve with that value and if you throw an error inside an async function it will reject the Promise with the error object. 
```

Comment:
`inside a_n_ async function` (missing n)

The `inside an async function` when you throw an error feels redundant, it should be clear that the async function is meant.
Same for `reject the Promise with the error object`, this could be `reject with the thrown error.` or something similar. (Especially `object` feels wrong here)